people/people.html

```diff
@@ -110,6 +110,17 @@ <h4 class="media-heading">Meriem Ferroudj</h4>
                 environmental sound.
             </div>
         </li>
+<li class="media">
+            <span class="pull-left">
+        ../images/jason.jpg"       <img class="media-object" src="" alt="Yvonne Phillips">
```

Comment:
The `../images/jason.jpg` string is outside of the html tag. This will render as plain text, rather then telling the browser to load an image. 

Instead move that string, into the `src` attribute, like so: `src="../images/jason.jpg"`